modules/autocomplete/main/index.coffee

```diff
@@ -131,7 +131,7 @@ buildAutoComplete = (searchTerm, fromCallback, loading) ->
     if not filteredData?
       message.text = @options.loadingMessage
     else if searchTerm.length < @options.minLength
-      message.text = @options.minLengthMessage.replace('$minLength', @options.minLength)
+      message.text = @options.pleaseEnterMinCharactersMessage.replace('$minLength', @options.minLength)
```

Comment:
Checked, it was added as part of the update to userFacingText:
https://github.com/ocadotechnology/hexagonjs/pull/220/files#diff-d62945ddda1e2d4c4438d9a9defcb6c8R134

The option added in the function did not match the value in the constructor... Probably best to fix it this way than to swap the constructor argument